packages/cli/yaml/validator/src/rules/valid-pagination/validateOffsetPagination.ts

```diff
@@ -151,23 +152,22 @@ function validateHasNextPageProperty({
     if (offsetPagination["has-next-page"] == null) {
```

Comment:
nit: A variable would be nice for this since we use `offsetPagination["has-next-page"]` in a few places here.

We previously referred to them as `propertyComponents`, but we could probably just do `path` so it's consistent with the new `validatePropertyInType` signature.